### PR TITLE
Add react-native-webview plugin

### PR DESCRIPTION
**Pull request by Karthiccc23**
Tested with sample miniapp here - https://github.com/Karthiccc23/testrnwebview-miniapp

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -1905,6 +1905,7 @@
       "react-native-video@4.4.5",
       "react-native-view-overflow@0.0.5",
       "react-native-view-shot@3.1.2",
+      "react-native-webview@11.26.1",
```
Comment (friederbluemle) [inline manifest.json:1908]: Most likely this should work with at least a few prior version as well, right?
If yes, can you add the manifest entry to 0.45-0.49 too?

**Review by friederbluemle — APPROVED** [review]
Thanks @Karthiccc23 👍

File: manifest.json (hunk 1 above)
Comment (Karthiccc23) [reply manifest.json:1908]: cool. But I have added the plugin in 0.46, should I move it to 0.45 and add support from 0.45-0.49 or just 0.46-0.49 is good?

Comment (Karthiccc23) [reply manifest.json:1912]: done

**Pull request merged**